package.json

```diff
@@ -111,6 +111,7 @@
     "react-content-loader": "^4.2.1",
     "react-dom": "^16.8.3",
     "react-modal": "^3.8.1",
+    "react-paginate": "^6.3.0",
```

Comment:
**Nitpick**

- Do you think we can implement the pagination algorithm without an extra plugin. Remember each plugin we install affects the size of the bundle.js file generated by webpack, and this file is served to the client.